src/adapters/WebexJSONAdapter.js

```diff
@@ -0,0 +1,13 @@
+import {ActivitiesJSONAdapter, PeopleJSONAdapter, RoomsJSONAdapter} from './';
+
+export default class WebexJSONAdapter {
+  /**
+   * Creates a new instance of the WebexAdapter.
+   * @param {Object} datasource The primary datasource the json adapter will be using.
```

Comment:
Can we add the required properties?

```suggestion
   * @param {Object} datasource The primary datasource the json adapter will be using.
   * @param {Object} datasource.activities
   * @param {Object} datasource.people
   * @param {Object} datasource.rooms
```

Reply:
I addressed your concern differently. Please take a look and let me know what you think. I prefer to not add an additional param, because that's not what the code does. I did however try to document the structure of our WebexJSONAdapter.